src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -12009,6 +12009,27 @@ void CEEJitInfo::allocMem (
         COMPlusThrowHR(CORJIT_OUTOFMEM);
     }
 
+    if (EventEnabledMethodJitMemoryAllocatedForCode())
+    {
+        SString methodBeingCompiledNames[3];
+        MethodDesc* methodBeingCompiled = m_pMethodBeingCompiled;
+        if (methodBeingCompiled)
+        {
+            (methodBeingCompiled)->GetMethodInfo((methodBeingCompiledNames)[0], (methodBeingCompiledNames)[1], (methodBeingCompiledNames)[2]);
+        }
+        else
+        {
+            (methodBeingCompiledNames)[0].Set(W("<null>"));
+            (methodBeingCompiledNames)[1].Set(W("<null>"));
+            (methodBeingCompiledNames)[2].Set(W("<null>"));
+        }
+
+        FireEtwMethodJitMemoryAllocatedForCode(methodBeingCompiledNames[0].GetUnicode(),
```

Comment:
Currently, coldCodeSize is required to be zero. But maybe this function should take `codeSize` (the sum of hot and cold code size) instead of just `hotCodeSize`, to be future-proofed, in case we eventually do allow hot/cold splitting for JIT?

Reply:
Sounds good. I forgot to ask you about this - but yes, I was wondering why `coldCodeSize` is always zero.